cli/wizard/commands/list-squid-token/index.ts

```diff
@@ -115,69 +115,95 @@ export async function listSquidToken() {
   const tokenListPath = path.resolve(process.cwd(), ...relativePath);
 
   await spinner("Creating PR...", async () => {
-    await $`git checkout -b feat/add-${newTokenConfig.symbol}-token`;
+    await $`git checkout -b feat/add-${newTokenConfig.prettySymbol}-token`;
     await $`git add ${tokenListPath}`;
-    await $`git commit -m "feat: add ${newTokenConfig.symbol} token"`;
+    await $`git commit -m "feat: add ${newTokenConfig.prettySymbol} token"`;
     await $`git push -u origin HEAD`;
   });
 }
 
 export type InterchainTokenInfo = {
   tokenId: string;
   tokenAddress: string;
+  tokenManagerAddress: string;
+  tokenManagerType: string;
   isOriginToken: boolean;
   isRegistered: boolean;
   chainId: number;
   axelarChainId: string;
   chainName: string;
   wasDeployedByAccount?: boolean;
-  kind: "canonical" | "standardized";
+  kind: "canonical" | "interchain" | "customInterchain";
 };
 
 export type InterchainTokenSearchResult = InterchainTokenInfo & {
   matchingTokens: InterchainTokenInfo[];
 };
 
-export type RemoteInterchainToken = {
+export type RemoteInterchainTokenApiResponse = {
+  name: string;
+  symbol: string;
   axelarChainId: string;
   tokenAddress: string;
+  tokenManagerAddress: string;
+  tokenManagerType: string;
   deploymentStatus: string;
   deploymentTxHash: string;
 };
 
-export type InterchainTokenDetails = {
+export type InterchainTokenDetailsApiResponse = {
   kind: string;
   salt: string;
   tokenName: string;
   tokenSymbol: string;
   tokenDecimals: number;
   tokenAddress: string;
+  tokenManagerAddress: string;
+  tokenManagerType: string;
+  originalMinterAddress: string;
   axelarChainId: string;
   tokenId: string;
-  deploymentTxHash: string;
-  deployerAddress: string;
-  remoteTokens: RemoteInterchainToken[];
+  deploymentMessageId: string;
+  deployer: string;
+  remoteTokens: RemoteInterchainTokenApiResponse[];
```

Comment:
```suggestion
  chains: RemoteInterchainTokenApiResponse[];
```
And make this be a map type?

Reply:
Could change this later if it's more involved to change the portal API

Reply:
yeah it's the API response that's `remoteTokens`